tests/xtro-sharpie/MacCatalyst-SceneKit.ignore

```diff
@@ -0,0 +1,8 @@
+# deprecated or not used on catalyst
+!missing-protocol-member! SCNProgramDelegate::programIsOpaque: not found
+!missing-selector! SCNMaterialProperty::borderColor not bound
+!missing-selector! SCNMaterialProperty::setBorderColor: not bound
+!missing-selector! SCNRenderer::render not bound
+
+# adding abstract member to an existing type would be a breaking change
+!incorrect-protocol-member! SCNAnimatable::removeAllAnimationsWithBlendOutDuration: is REQUIRED and should be abstract
```

Comment:
when something needs to be annotated for all platforms then you should add it to the `common-*.ignore` file

Reply:
got it, thank you!